frontend/README.md

```diff
@@ -1,68 +1,50 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Headlamp Frontend
```

Comment:
replaced a placeholder CRA readme, this duplicates a bit what we have in the docs and we would need to consolidate this somehow but not in scope of this pr